apps/dashboard/app/apps/ood_app.rb

```diff
@@ -181,12 +181,26 @@ def manifest_path
     path.join("manifest.yml")
   end
 
-  def icon_path
-    path.join("icon.png")
+  def svg_icon?
+    path.join("icon.svg").file?
+  end
+
+  def png_icon?
+    path.join("icon.png").file?
   end
```

Comment:
Can we memorize these? seems like could be a lot of file reads. twice as many as before anyhow.